packages/electron-publish/src/BintrayPublisher.ts

```diff
@@ -72,6 +72,7 @@ export class BintrayPublisher extends HttpPublisher {
             "Content-Length": dataLength,
             "X-Bintray-Override": "1",
             "X-Bintray-Publish": "1",
+            "X-Bintray-Debian-Architecture": arch
```

Comment:
See https://github.com/electron-userland/electron-builder/pull/1885 for a WIP of the other headers.